Clockwork/Support/Vanilla/config.php

```diff
@@ -13,7 +13,7 @@
 	|
 	*/
 
-	'enable' => isset($_ENV['CLOCKWORK_ENABLE']) ? $_ENV['CLOCKWORK_ENABLE'] : true,
+	'enable' => getenv('CLOCKWORK_ENABLE') !== false ? getenv('CLOCKWORK_ENABLE') : false,
```

Comment:
By default was enabled, now it's not. It is not obvious in this commit, may be this is a mistake?

Reply:
That was a mistake, I'm sure.